shared/metadata.go

```diff
@@ -4,6 +4,38 @@
 
 package shared
 
+import (
+	"net/http"
+	"path"
+	"sort"
+
+	"github.com/go-yaml/yaml"
+	"github.com/web-platform-tests/wpt-metadata/util"
+)
+
+// MetadataKey determines whether Metadata Information returns along
+// with a test result query request.
+const MetadataKey = "metadataInfo"
+
+// MetadataResponse is a response to a wpt-metadata query.
+type MetadataResponse struct {
```

Comment:
Again, what is the purpose of this struct type? Can't `MetadataResult` simply be of type `shared.MetadataResults`?